0071-boring-protocol.md

```diff
@@ -0,0 +1,269 @@
+
+<h2> Stakeholders </h2>
+                                                                                                           
+This proposal impacts all current and future participants in the Helium Ecosystem as well as all current BOP holders.
+
+The objective is to provide greater growth to both networks while equally providing opportunity to the economics of Helium hardware operators, simultaneously creating privacy access to Helium network infrastructure. The Helium token model is described in the HIP’s presented previous to this proposal. 
+
+The BOP token utility is mainly a settlement between “clients” of the dVPN and “operators” of nodes which are used to privately access the internet. Apart from this primary function the token is staked by node operators to present loyalty to the network and financial incentive to act in “good faith” as they operate exit nodes. As nodes remain connected they increase their opportunity for traffic/data to be directed through their node and increase their profitability as the token is settled on a data consumption rate.
+
+Continued participation and discussion is greatly encouraged to ensure the proposal is in line with the needs of both communities.
+                                                                                                           
+## Motivation
+
+The drivers for this proposal are as follows:
+
+-   Privacy and Security for Helium “subscribers” and internet users as a whole
+-   Security and Scalability of the L1 as proposed in HIP-51
+-   Additional mechanisms for profitability to Helium Network infrastructure operators
+-   Proving a model of multi chain redundancy to further secure network economics and participant profitability
+-   Sustainability in manufacturing and distribution of hardware
+-   Increased Utility and Security for EVM mechanisms like wHNT
+
+A dVPN implementation would be an immediate and relevant re-purpose,  of the no longer needed processing power of ~1 million LoRa Hotspots sitting online, into the largest dVPN network right out of the box. Which provides the customer ability to switch client/ server modes in-app and add the dVPN service and BOP/dVPN token as another subDAO along with IoT and MOBILE. 
+
+The Boring Protocol code should be able to be added as a firmware update to run along the kernel in a container similar to the LoRa packet processing and forwarder. The existing hotspot network is ideal for supporting this type of tunneling traffic—as most hotspots have gigabit ethernet connections with private IPs and have adequate bandwidth, processing power and storage already. The RAK and MNTD are based on PI4s and should make a perfect mirror to test run in our distributed environment.
+                                                                                                           
```

Comment:
can you attach the boring protocol code, please?

Reply:
Link added to detailed description. All relevant code to be built in the open and included before draft is complete.